### PR TITLE
#163281277 Integrates database to Travis and adds script to auto-generate tables

### PR DESCRIPTION
**Pull request by emdeechege**
### Title 
Setup database integration with Travis and add scripts to auto-generate tables

### Description 
* Add additional seetings to Travis to link with Postgres
* Create a local database for the project
* Create test-database for the project
*  scripts to run the db

### How should this be manually tested?
* Clone this repository
* Cd into the folder
* Create a virtual environment:
* Virtualenv -p python3 venv
*  Pip install -r requirements.txt
* Run python manage.py to create the tables
* Read through Travis file

### Pivotal Tracker
https://www.pivotaltracker.com/n/projects/2235156/stories/16328127

### CODE REVIEW

File: app/connect.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+from app import schema
+from instance.config import Config
+
+
+
+def init_db():
+    """ setup database connection"""
+    url = os.getenv('DATABASE_URL')
+    conn = psycopg2.connect("dbname=questioner user=emdee password=arif@123")
+    conn.commit()
```
Comment (RuthNjeri) [inline app/connect.py:13]: If you are not using the url from the database environment you should remove it

File: app/api/v2/views/questions_views.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        postedBy, meetup_id, title, content)
+
+    return question
+
+
+@v1_question_blueprint.route('/questions', methods=['GET'])
+def getall():
+    """ endpoint to fetch all questions """
+
+    data = questions.getall_questions()
```
Comment (RuthNjeri) [inline app/api/v2/views/questions_views.py:38]: rename the variable data to be more descriptive according to the contents it contains

File: app/connect.py (hunk 1 above)
Comment (emdeechege) [reply app/connect.py:13]: I have removed the unused variables

File: app/api/v2/views/questions_views.py (hunk 2 above)
Comment (emdeechege) [reply app/api/v2/views/questions_views.py:38]: adjusted as required

**Pull request merged**